src/table/resizer/index.tsx

```diff
@@ -42,10 +49,12 @@ export function Resizer({ onDragMove, onFinish }: ResizerProps) {
     };
 
     const updateColumnWidth = (newWidth: number) => {
+      const { right, width } = headerCell.getBoundingClientRect();
+      const updatedWidth = newWidth < minWidth ? minWidth : newWidth;
+      updateTrackerPosition(right + updatedWidth - width);
+      setHeaderCellWidth(newWidth);
       // callbacks must be the last calls in the handler, because they may cause an extra update
       onDragStable(newWidth);
-      // we read the element size again because the previous callback changes it
-      updateTrackerPosition(headerCell.getBoundingClientRect().right);
```

Comment:
onDragStable update the column width with useState. At this moment the width can be not yet updated. It should be updated by known width value change, which is more reliable.